tests/ImageSharp.Tests/Image/ImageTests.WrapMemory.cs

```diff
@@ -282,6 +282,76 @@ public void WrapSystemDrawingBitmap_FromBytes_WhenObserved()
                 }
             }
 
+            [Fact]
+            public unsafe void WrapMemory_FromPointer_CreatedImageIsCorrect()
+            {
+                var cfg = Configuration.CreateDefaultInstance();
+                var metaData = new ImageMetadata();
+
+                var array = new Rgba32[25];
+
+                fixed (void* ptr = array)
+                {
+                    using (var image = Image.WrapMemory<Rgba32>(cfg, ptr, 5, 5, metaData))
+                    {
+                        Assert.True(image.TryGetSinglePixelSpan(out Span<Rgba32> imageSpan));
+                        ref Rgba32 pixel0 = ref imageSpan[0];
+                        Assert.True(Unsafe.AreSame(ref array[0], ref pixel0));
+                        ref Rgba32 pixel_1 = ref imageSpan[imageSpan.Length - 1];
+                        Assert.True(Unsafe.AreSame(ref array[array.Length - 1], ref pixel_1));
+
+                        Assert.Equal(cfg, image.GetConfiguration());
+                        Assert.Equal(metaData, image.Metadata);
+                    }
+                }
+            }
+
+            [Fact]
+            public unsafe void WrapSystemDrawingBitmap_FromPointer_WhenObserved()
```

Comment:
```suggestion
            public unsafe void WrapSystemDrawingBitmap_FromPointer()
```

Nit: "observation" is a concept related to `Memory<T>` and `IMemoryOwner`, not applicable here.

Unless it ends up with a terrible code bloat, I would also try to inline the locking from `BitmapMemoryManager` to make it more clear what are we testing.

Reply:
What do you mean with "inline the locking"? I mean, I thought the point here was to showcase how you could have a self-contained memory manager wrapping a `System.Drawing` image and controlling its lifetime 🤔

Reply:
I meant inlining the *logic*. Today was a day of terrible typos.

Reply:
I mean: this is the case when you dont have MemoryManager. If you had one, you could just pass it (or the Memory).

Reply:
But this is really just a nit.

Reply:
Well I got busy with the other optimizations and James merged this PR before I could do this 😆
Let me know if you want me to create another PR to refactor this bit! 🙂

Reply:
Feel free to but only if you have the time.